frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -59,13 +78,18 @@ const exportStatisticsToCSV = (
           ...row.boundingBoxTopLeft,
           ...row.boundingBoxPosition,
         ]
+          .filter((el) => el != null && el.toString().length > 0)
```

Comment:
is this meant to remove additional coordinates that could be null? I think, this filtering is dangerous as it can easily remove other columns (for example, a segment name could be an empty string) which would result in an invalid CSV.
How about conditionally adding `row.additionalCoordinates` to the values array above if `hasAdditionalCoords` is true?